packages/@react-stately/numberfield/src/useNumberFieldState.ts

```diff
@@ -190,11 +200,13 @@ export function useNumberFieldState(
     incrementToMax,
     decrement,
     decrementToMin,
+    canIncrement,
+    canDecrement,
     minValue,
     maxValue,
-    numberValue,
+    numberValue: parsedValue,
```

Comment:
just to make sure I'm reading it right, the numberValue will not be the same value as someone might have from listening to onChange?

Reply:
that is correct